lib/inventory_sync/async/inventory_scheduled_sync.rb

```diff
@@ -0,0 +1,19 @@
+module InventorySync
+  module Async
+    class InventoryScheduledSync < ::Actions::EntryAction
+      include ::Actions::RecurringAction
+
+      def plan
+        concurrence do
```

Comment:
concurrence is the default flow so you can drop this block and reduce the nesting by one level